AirCasting/SessionViews/ChartViewModel.swift

```diff
@@ -28,17 +28,19 @@ final class ChartViewModel: ObservableObject {
     private let numberOfEntries = Constants.Chart.numberOfEntries
     
     private var backgroundNotificationHandle: Any?
+    private var useCelsius: Bool
     
     deinit {
         mainTimer?.invalidate()
         firstTimer?.invalidate()
     }
     
-    init(session: SessionEntity, persistence: PersistenceController) {
+    init(session: SessionEntity, persistence: PersistenceController, useCelsius: Bool) {
```

Comment:
not a fan of passing a flag here too, but this would require a solid rewrite so maybe let's pass.

But... I think we'll have to start doing such refactorings along the way soon in order to clean the code up a bit. Let's wait for injection mechanism to be merged in first tho.